packages/node-bridge/tsconfig.json

```diff
@@ -2,11 +2,13 @@
   "compilerOptions": {
     "allowJs": true,
     "checkJs": true,
-    "lib": ["esnext"],
+    "lib": ["ES2020"],
     "noEmit": true,
     "noImplicitReturns": true,
     "strict": true,
-    "declaration": true
+    "target": "ES2020",
+    "declaration": true,
+    "module": "ES2020"
```

Comment:
I'm not sure this is correct. The `module` should likely be `commonjs` since we even changed TS files to JS in this package to preserve CJS

Reply:
Yeah, not sure how this change made it in. I removed it. [`8edba39` (#7865)](https://github.com/vercel/vercel/pull/7865/commits/8edba39fa5ac28d21ebf43c0cf335e723f241b91)